tests/queries/dune_cow_rewards.sql

```diff
@@ -0,0 +1,37 @@
+with
```

Comment:
Notice that this was the dune query removed from the `cow_rewards.sql`. We are using this now to test the orderbook query and Dune agree.